### ISSUE TITLE
Submit to vim

### ISSUE THREAD

**Issue by coot** [purescript-contrib/purescript-vim#50]
It would be a good idea to submit the files to vim.

**Comment by coot** [comment]
This will require re-licensing using vim charity license.

**Comment by wsdjeg** [comment]
I do not think we need to submit to vim runtime, BTW, maybe submit a basic version. use github repo for develop version will make it easier to hot fix bug and implement new feature.

**Comment by andys8** [comment]
Vim is [aware of many file types](https://github.com/vim/vim/tree/master/runtime/ftplugin), but PureScript is missing. Because there aren't many new features, I'd be in favour to submit it to [Vim](https://github.com/vim/vim)
